src/MOIWrapper.jl

```diff
@@ -368,3 +368,13 @@ end
 function LQOI.delete_variables!(instance::Optimizer, start_col::Int, end_col::Int)
     delete_columns(instance.inner, [Cint(i-1) for i in start_col:end_col])
 end
+
+function LQOI.add_mip_starts!(instance::Optimizer, cols::Vector{Int},
+                              x::Vector{Float64})
+
+    input = zeros(Float64, get_num_cols(instance.inner))
+    for (col, val) in zip(cols, x)
```

Comment:
If one starting value is not given, it is silently set to zero. This isn't what MOI documents.
Is an error already thrown in LQOI and cols is guaranteed to contain all variables ?